drivers/display/ssd1327.c

```diff
@@ -67,186 +81,256 @@ static inline int ssd1327_write_bus_cmd(const struct device *dev, const uint8_t
 	return 0;
 }
 
-static inline int ssd1327_set_timing_setting(const struct device *dev)
+static inline int ssd1327_write_bus_cmd_i2c(const struct device *dev, const uint8_t cmd,
+					    const uint8_t *data, size_t len)
 {
 	const struct ssd1327_config *config = dev->config;
-	uint8_t buf;
+	static uint8_t buf[SSD1327_MAXIMUM_CMD_LENGTH];
```

Comment:
static keyword should be dropped?

Reply:
Why? It prevents doing a allocation on stack everytime this function is called there

Reply:
But what happens if user has multiple instances of the display on different buses? Access to the buffer is not thread-safe?

Reply:
Yes... It is not thread-safe. Will update.

Reply:
Done